base/src/org/compiere/process/MigrationFromXMLAbstract.java

```diff
@@ -46,6 +46,8 @@
 	@Override
 	protected void prepare() {
 		filePathOrName = getParameterAsString(FILEPATHORNAME);
+		if (filePathOrName == null)
```

Comment:
Hi @e-Evolution it is good for work but bad for future changes, this class is volatile and any developer can regenerate from Application Dictionary